charts/yourls/templates/NOTES.txt

```diff
@@ -89,19 +89,15 @@ You can access Apache Prometheus metrics following the steps below:
 {{- include "common.warnings.rollingTag" .Values.image }}
 {{- include "common.warnings.rollingTag" .Values.metrics.image }}
 {{- include "common.warnings.rollingTag" .Values.volumePermissions.image }}
-{{- $passwordValidationErrors := list -}}
```

Comment:
The password validation array should stay outside of the actual validation for maintainability.